Bugsnag/Payload/BugsnagHandledState.m

```diff
@@ -199,6 +206,7 @@ + (SeverityReasonType)severityReasonFromString:(NSString *)string {
 - (NSDictionary *)toJson {
     NSMutableDictionary *dict = [NSMutableDictionary new];
     dict[kUnhandled] = @(self.unhandled);
+    dict[kUnhandledOverridden] = @(self.unhandledOverridden);
```

Comment:
This is specified in the product design to default to `false` if omitted, so a check here to avoid sending a few extra bytes seems reasonable.